src/commands/speak.ts

```diff
@@ -0,0 +1,15 @@
+import { Message } from 'discord.js'
+
+export async function speak(message: Message) {
+    console.log('here')
```

Comment:
Where is here? why are we here? so many questions...

Reply:
my bad